test/Microsoft.AspNet.Mvc.TagHelpers.Test/ImageTagHelperTest.cs

```diff
@@ -0,0 +1,265 @@
+            };
+
+            // Act
+            helper.Process(context, output);
+
+            // Assert
+            Assert.True(output.Content.IsEmpty);
+            Assert.Equal("img", output.TagName);
+            Assert.Equal(2, output.Attributes.Count);
+            TagHelperAttribute srcAttribute = Assert.Single(output.Attributes, attr => attr.Name.Equals("src"));
+            Assert.Equal("/images/test-image.png?v=f4OxZX_x_FO5LcGBSKHWXfwtSx-j1ncoSt3SABJtkGk", srcAttribute.Value);
+        }
+
+        [Fact]
+        public void RendersImageTag_AddsFileVersion_WithRequestPathBase()
+        {
+            // Arrange
+            var context = MakeTagHelperContext(
+                attributes: new TagHelperAttributeList
+                {
+                    { "alt", new HtmlString("alt text") },
+                    { "src", "/bar/images/image.jpg" },
+                    { "asp-file-version", "true" },
+                });
+            var output = MakeImageTagHelperOutput(attributes: new TagHelperAttributeList
+            {
+                { "alt", new HtmlString("alt text") },
+            });
+            var hostingEnvironment = MakeHostingEnvironment();
+            var viewContext = MakeViewContext("/bar");
+            var helper = new ImageTagHelper
+            {
+                HostingEnvironment = hostingEnvironment,
+                ViewContext = viewContext,
+                Src = "/bar/images/image.jpg",
+                FileVersion = true,
+                Cache = MakeCache(),
+            };
+
+            // Act
+            helper.Process(context, output);
+            // Assert
+            Assert.True(output.Content.IsEmpty);
+            Assert.Equal("img", output.TagName);
+            Assert.Equal(2, output.Attributes.Count);
+            TagHelperAttribute srcAttribute = Assert.Single(output.Attributes, attr => attr.Name.Equals("src"));
+            Assert.Equal("/bar/images/image.jpg?v=f4OxZX_x_FO5LcGBSKHWXfwtSx-j1ncoSt3SABJtkGk", srcAttribute.Value);
+        }
+
+        private static ViewContext MakeViewContext(string requestPathBase = null)
+        {
+            var actionContext = new ActionContext(new DefaultHttpContext(), new RouteData(), new ActionDescriptor());
+            if (requestPathBase != null)
+            {
+                actionContext.HttpContext.Request.PathBase = new Http.PathString(requestPathBase);
+            }
+
+            var metadataProvider = new EmptyModelMetadataProvider();
+            var viewData = new ViewDataDictionary(metadataProvider);
+            var viewContext = new ViewContext(
+                actionContext,
+                Mock.Of<IView>(),
+                viewData,
+                Mock.Of<ITempDataDictionary>(),
+                TextWriter.Null);
+
+            return viewContext;
+        }
+
+        private static TagHelperContext MakeTagHelperContext(
+            TagHelperAttributeList attributes = null,
+            string content = null)
```

Comment:
Simplify method signature: No need for `attributes` to be optional.  No need for `content` at all.